src/js/controller/PaletteController.js

```diff
@@ -106,11 +110,21 @@
     } else {
       colorPicker.spectrum('set', color);
     }
-    this.setTitleOnPicker_(color, colorPicker);
   };
 
-  ns.PaletteController.prototype.setTitleOnPicker_ = function (title, colorPicker) {
-    var spectrumInputSelector = '.sp-replacer';
-    colorPicker.next(spectrumInputSelector).attr('title', title);
+  /**
+   * @private
+   */
+  ns.PaletteController.prototype.showValueDisplayer_ = function (evt) {
```

Comment:
How about updating the content of the colopicker tooltip instead. It's already there so no extra UI needed. We're on a somewhat old version of the jquery tooltips, but updating is possible.

$(element).attr('data-original-title', newValue)
          .tooltip('fixTitle');

(tooltips are attached to the parent of "#color-picker")